### ISSUE TITLE
Request: ATGames Legends Gamer Pro

### ISSUE THREAD

**Issue by darthnater4ever** [ndeadly/MissionControl#171]
Hey, just leaving this as a reminder from the gbatemp forum. Could you add support [this](https://www.atgames.net/arcades/legends-gamer-series/) controller once Bluetooth LE is working? In the meantime I could attempt to get any information for the controller if needed.

**Comment by ndeadly** [comment]
Thanks, I'll be sure to get back to this one LE support is ready.
There's probably not a lot I need from you right now, but if you like you could use a mobile app like nRF Connect to get a log of all the exposed BLE characteristics to verify it presents itself as a standard HID device. It's nothing urgent though.
Maybe you could also tell me what might make a sensible default button mapping from the controller to a pro controller layout. I'm not familiar with arcade style controllers and whether there's any kind of standard button layout or whether you typically assign your own based on the game you're playing.

**Comment by darthnater4ever** [comment]
For the button layout I'm not sure yet because I have no idea which buttons are on which controller as I normally have both connected at once. On the atgame console the paddle buttons mirror some face buttons so I don't know if they're their own buttons or not. 

The joystick should most likely emulate a thimbstick if possible, I'm pretty sure it's just 8 direct but don't know what software to test the controllers with.

Also, I believe I got the information from the nRF app. 

<details><summary>P1 Deck</summary>

  <details>

  <summary>Client</summary>

![20210103_205841](https://user-images.githubusercontent.com/76895502/103504532-8ef61c80-4e0c-11eb-929a-330fba88c794.jpg)
</details>

<details>

  <summary>Server</summary>

![20210103_205927](https://user-images.githubusercontent.com/76895502/103504540-93223a00-4e0c-11eb-844d-500c9b62a657.jpg)
</details>

[Log 2021-01-03 21_27_38.txt](https://github.com/ndeadly/MissionControl/files/5763216/Log.2021-01-03.21_27_38.txt)
</details>

<details>
  <summary>P2 Deck</summary>

<details>

  <summary>Client</summary>

![20210103_205959](https://user-images.githubusercontent.com/76895502/103504545-96b5c100-4e0c-11eb-9819-cc68b9907487.jpg)
</details>

<details>

  <summary>Server</summary>

![20210103_210014](https://user-images.githubusercontent.com/76895502/103504554-99b0b180-4e0c-11eb-91b7-7ac81984b775.jpg)
</details>

[Log 2021-01-03 21_27_44.txt](https://github.com/ndeadly/MissionControl/files/5763217/Log.2021-01-03.21_27_44.txt)
</details>

**Comment by Nilaoba12345** [comment]
Is this solved in the latest update? I do see “AtGames Legends Pinball Controller” on the support list but when I tried it it’s just blinking in pairing mode. I can confirm my switch is working fine with ps4 controllers so it has to be the controller. I also have a atgames legend gamer pro with 2 players in one deck just like the above. Can provide details if need to but I suspect it will be the same as what has been provided above.

**Comment by ndeadly** [comment]
No, this is a different controller that uses Bluetooth LE. That's still not supported as of 0.7.0. The controller listed as working is a different model that uses the older Bluetooth standard.
